stdlib/Random/docs/src/index.md

```diff
@@ -283,8 +283,10 @@ Note: `Sampler(rng, x)` is simply a shorthand for `Sampler(rng, x, Val(Inf))`, a
 The API is not clearly defined yet, but as a rule of thumb:
 1) any `rand` method producing "basic" types (`isbitstype` integer and floating types in `Base`)
    should be defined for this specific RNG, if they are needed;
-2) other documented `rand` methods accepting an `AbstractRNG` should work out of the box,
-   (provided the methods from 1) what are relied on are implemented),
+2) make sure that `copy` is possible for the new generator, otherwise, (i.e. hardware-based RNG ),
+   an error should be thrown;
```

Comment:
I suggest instead: "make sure that `copy` is implemented for the new generator when possible, otherwise (e.g. hardware-based RNG), an error should be thrown."

But I'm not convinced that a specific error should be thrown, I believe that the default `MethodError` is enough.
So another way to put it is simply: "`copy` for RNGs should be implemented when possible."
I will let @tpapp comment on whether he thinks more information should be provided.

Reply:
Thanks, I got it. Please let me know if there is something I can do for this.